.env.dist

```diff
@@ -27,6 +27,11 @@ MAIL_FROM_PREFIX=
 MAIL_USER=
 MAIL_PASS=
 
+MAIL_BLOB_STORAGE_URL=
```

Comment:
@morgang-adappt 

similar question for this
It appears it's currently used
but do you have more information about that? What values can I use?

Reply:
Hi @AntoineRelief,

These are the values used to persist images linked in the email (header, footer, and banner logos). At the moment we have been using a container in the ADAPPT-POC RG with container level access. From looking at the storage accounts in the EMSSAFE-D RG, it seems it should be possible to create an `email-images` (or similar) container for use, and use the appropriate endpoint, account name, access key, and container name.